src/wdio/browser.ts

```diff
@@ -127,4 +127,17 @@ export class Browser {
         const caps = browser.capabilities as any;
         return caps.platformName;
     }
+
+    // Window
+    public static async getWindowSize(): Promise<void> {
```

Comment:
Same question about the getter returning void as @joaquimvila denoted in the other PR

Reply:
Updated accordingly